packages/server/src/utilities/rowProcessor/tests/utils.spec.ts

```diff
@@ -1,11 +1,29 @@
-import { fixAutoColumnSubType } from "../utils"
+import { fixAutoColumnSubType, processAIColumns } from "../utils"
 import { AutoFieldDefaultNames } from "../../../constants"
 import {
+  AIOperationEnum,
   AutoFieldSubType,
   FieldSchema,
   FieldType,
+  INTERNAL_TABLE_SOURCE_ID,
   RelationshipType,
+  Table,
+  TableSourceType,
 } from "@budibase/types"
+import { generator } from "@budibase/backend-core/tests"
+
+const buildPromptMock = jest.fn()
+
+jest.mock("@budibase/pro", () => ({
```

Comment:
It is a good practice to type the mocked interface, so it can be useful when refactoring or extending libraries. See `packages/server/src/utilities/rowProcessor/tests/bbReferenceProcessor.spec.ts`